recipe/meta.yaml

```diff
@@ -26,13 +26,14 @@ requirements:
   host:
     - bzip2
     - glib
-    - libxml2 2.10
+    - libxml2 {{ libxml2 }}
     - pcre 8.45
     - zlib 1.2.13
+    - gettext  # [osx]
   run:
     - bzip2
     - glib
-    - libxml2 >=2.10.3,<2.11.0a0
+    - libxml2 >=2.13.1
```

Comment:
pin_compatible would probably be better. Also here there's no upper bound, is that intentional?